features/steps/flaw_detail.py

```diff
@@ -445,6 +445,25 @@ def step_impl(context):
     flaw_detail_page.click_btn('saveBtn')
     flaw_detail_page.wait_msg('flawSavedMsg')
 
+
 @then("The embargoed flaw is updated")
 def step_impl(context):
     context.browser.quit()
+
+
+@when("I add a new affect with valid data")
+def step_impl(context):
+    go_to_specific_flaw_detail_page(context.browser)
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.click_button_with_js('addNewAffectBtn')
+    context.ps_component = flaw_detail_page.set_new_affect_inputs()
+    flaw_detail_page.click_btn('saveBtn')
+    flaw_detail_page.wait_msg('affectCreatedMsg')
```

Comment:
Does the adding affect work well on the current stage?
I tried to run the feature for CVE-2023-2023, but it gave the following error. I went to the selenium page, I saw the saving has been running...
 _seleniumpagefactory.Pagefactory.ElementNotFoundException: An exception of type TimeoutException occurred. With Element -: affectCreatedMsg - locator: (xpath, //div[text()='Affect Created.'])_

Reply:
After discussing off line, this is caused by the test data.